_episodes/02-filedir.md

```diff
@@ -157,12 +158,44 @@ Desktop/      Downloads/    Movies/       Pictures/
 
 Here,
 we can see that our home directory contains mostly **sub-directories**.
-Any names in your output that don't have trailing slashes,
+Any names in your output that don't have a classification symbol,
```

Comment:
For lines 139-161, I somewhat prefer the old wording (modulo the fixes you made to the repetition of "option", and the no-longer-applicable "earlier example" callback), because:

* A typical learner is unlikely to be familiar with executable files (in the unix sense), and even less likely to be familiar with links. 
* Calling it the "classify option" seems a little confusing, because it's not obvious that `-F` would be an abbreviation for "classify". A reader might even think that "classify option" is some generic piece of shell jargon? You could try to introduce the `--classify` long form, though I don't know if that would be helpful at this point.

Reply:
Hmm.
I was trying to address [this comment](https://github.com/swcarpentry/shell-novice/issues/644#issuecomment-374961770). 

How about this?
> We can make its output more comprehensible by using the `-F` **option** 
(also known as a **switch** or a **flag**) ,